tasks.py

```diff
@@ -213,6 +213,19 @@ def poll(self, source):
                                                include_redirect_sources=False)
             activity['mentions'].update(u.get('value') for u in urls)
             responses[id] = activity
+            break
+
+      # handle quote mentions
+      for att in obj.get('attachments', []):
+        if (att.get('objectType') in ('note', 'article')
```

Comment:
i was suspicious at first that we needed this same logic in both places, but i get it now. lgtm.